### PR TITLE
[ADD] A new module that links products on sale order lines to purchase orders

### PR DESCRIPTION
**Pull request by difbit**
A new module that links products on sale order lines to corresponding purchase orders

### CODE REVIEW

File: sale_order_show_purchase_order/models/sale_order_line.py [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+
+        for record in self.order_line:
+            if record != False:
+                for line in self.order_line:
+                    if line.procurement_ids.ids != []:
+                        group_list.append(line.procurement_ids.group_id)
+
+            if group_list != []:
+                for group in group_list:
+                    print group
```
Comment (jarmokortetjarvi) [inline sale_order_show_purchase_order/models/sale_order_line.py]: Remove print

File: sale_order_show_purchase_order/models/sale_order_line.py [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+                    for id_number in group.procurement_ids:
+                        if id_number.purchase_id:
```
Comment (jarmokortetjarvi) [inline sale_order_show_purchase_order/models/sale_order_line.py]: Check for `ids` before trying to access `ids[0]`
`if id_number.purchase_id and if id_number.purchase_id.ids`:

**Review by jarmokortetjarvi — CHANGES_REQUESTED** [review]

File: sale_order_show_purchase_order/models/sale_order_line.py (hunk 1 above)
Comment (difbit) [reply sale_order_show_purchase_order/models/sale_order_line.py]: This print has been removed

File: sale_order_show_purchase_order/models/sale_order_line.py (hunk 2 above)
Comment (difbit) [reply sale_order_show_purchase_order/models/sale_order_line.py]: Changed the condition to 
"if id_number.purchase_id and id_number.purchase_id.ids:"
This should work too?

File: sale_order_line_placeholder_product/i18n/fi.po [hunk 3]
```diff
@@ -7,7 +7,11 @@ msgstr ""
 "Project-Id-Version: Odoo Server 10.0\n"
 "Report-Msgid-Bugs-To: \n"
 "POT-Creation-Date: 2019-04-08 08:14+0000\n"
+<<<<<<< HEAD
```
Comment (jarmokortetjarvi) [inline sale_order_line_placeholder_product/i18n/fi.po]: Fix conflict or re-create the translation file

File: sale_order_show_purchase_order/__manifest__.py [hunk 4]
```diff
@@ -0,0 +1,38 @@
...
+    'website': 'https://tawasta.fi',
+    'author': 'Oy Tawasta Technologies Ltd.',
+    'license': 'AGPL-3',
+    'application': False,
+    'installable': True,
+    'depends': [
+        'sale',
+        'purchase',
+    ],
+    'data': ['views/sale_order.xml'
```
Comment (jarmokortetjarvi) [inline sale_order_show_purchase_order/__manifest__.py]: Fix indentation

**Review by jarmokortetjarvi — CHANGES_REQUESTED** [review]

**Pull request merged**